File: Easy/LCE - 1304.js
/**
 * @param {number} n
 * @return {number[]}
 */
var sumZero = function(n) {
    
    let ans = [], count=1
    
    if(n%2)
        ans.push(0)
    n--
    
    while(n>0) {
        n -= 2
        ans = [count*-1, ...ans, count*1]
        count++
    }
    
    return ans
    
};

File: Easy/Python - Solutions/LCE_1304.py
class Solution:
    def sumZero(self, n: int) -> List[int]:
        if n == 1:
            return [0]

        ans = []
        if n % 2 == 0:
            count = n//2
            for _ in range(n+1):
                ans.append(count)
                count -= 1
            ans.remove(0)
        else:
            count = n//2
            for _ in range(n):
                ans.append(count)
                count -= 1

        return ans
